### PR TITLE
deactivate conditionals addition

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

This PR adds the functionality for a user to deactivate the downstream impacts of order adjustments and refunds.

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide explanation how the change is non breaking below.)

This PR adds a conditional block throughout the package which will be true by default. As such, this will not be a breaking change for existing users.

**Is this PR in response to a previously created Issue**

- [X] Yes, Issue [#23](https://github.com/fivetran/dbt_shopify_source/issues/23)
- [ ] No 

**How did you test the PR changes?** 

 
- [X] CircleCi  
- [ ] Other (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 
- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [X] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🧊 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,20 @@ vars:
...
 
+### Disable Models
+This package was designed with the intention that users have all relevant Shopify tables being synced by Fivetran. However, if you are Shopify user that does not operate on returns or adjustments then you will not have the related source tables. As such, you may use the below variable configurations to disable the respective downstream models. All variables are `true` by default, only add the below configuration if you are wishing to disable the models:
+
+```yml
+# dbt_project.yml
+
+...
+vars:
+  shopify_source:
```
Comment (fivetran-joemarkiewicz) [inline README.md:68]:
```suggestion
  shopify:
```

**Review by fivetran-joemarkiewicz — COMMENTED** [review]
one quick fix

File: README.md [hunk 2]
```diff
@@ -57,6 +57,20 @@ vars:
     union_databases: ['shopify_usa','shopify_canada'] # use this if the data is in different databases/projects but uses the same schema name
 ```
 
+### Disable Models
+This package was designed with the intention that users have all relevant Shopify tables being synced by Fivetran. However, if you are Shopify user that does not operate on returns or adjustments then you will not have the related source tables. As such, you may use the below variable configurations to disable the respective downstream models. All variables are `true` by default, only add the below configuration if you are wishing to disable the models:
```
Comment (fivetran-jamie) [inline README.md:61]:
```suggestion
This package was designed with the intention that users have all relevant Shopify tables being synced by Fivetran. However, if you are a Shopify user that does not operate on returns or adjustments then you will not have the related source tables. As such, you may use the below variable configurations to disable the respective downstream models. All variables are `true` by default. Only add the below configuration if you are wishing to disable the models:
```

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/shopify_source
-    version: [">=0.5.0", "<0.6.0"]
+  # - package: fivetran/shopify_source
+  #   version: [">=0.5.0", "<0.6.0"]
+  - git: https://github.com/fivetran/dbt_shopify_source.git
```
Comment (fivetran-jamie) [inline packages.yml:4]: obligatory reminder to update

**Review by fivetran-jamie — APPROVED** [review]
looks great! just a readme comment

File: README.md [hunk 4]
```diff
@@ -57,6 +57,20 @@ vars:
...
+
+```yml
+# dbt_project.yml
+
+...
+vars:
+  shopify:
+    shopify__using_order_adjustment:  false  # true by default
+    shopify__using_order_line_refund: false  # true by default
+    shopify__using_order_refund:      false  # true by default
```
Comment (fivetran-joemarkiewicz) [inline README.md:71]:
```suggestion
  shopify__using_order_adjustment:  false  # true by default
  shopify__using_order_line_refund: false  # true by default
  shopify__using_order_refund:      false  # true by default
```

**Pull request merged**